modules/src/dice.py

```diff
@@ -3,14 +3,12 @@
 from templates.text import TextTemplate
 from templates.quick_replies import add_quick_reply
 
-dice_sides = {
-	1: u'\u2680',
-	2: u'\u2681',
-	3: u'\u2682',
-	4: u'\u2683',
-	5: u'\u2684',
-	6: u'\u2685'
-}
+dice_sides = {1: 'https://upload.wikimedia.org/wikipedia/commons/6/62/Kismet-Ace.png',
```

Comment:
We prefer using images from http://fa2png.io/ wherever possible. They are quite elegant! 😄 
See http://fa2png.io/r/foundation-icon-fonts/die-one/?color=e74c3c&margin=0&size=256
Clicking on generate button yields http://fa2png.io/media/icons/foundation-icon-fonts/2015-02-16/die-one/256/0/e74c3c_none.png (see download button link)